ROADMAP.md

```diff
@@ -0,0 +1,54 @@
+# 2024 Roadmap
+
+The conda-store roadmap for the remaining part of 2024 into 2025 will revolve around several key areas of iterative improvement:
+
+- Performance
+- Backup and restore
+- User experience improvements
+
+| Mark | Description      |
+| ---- | ---------------- |
+| 📪   | work not started |
+| ✅   | work completed   |
+| ⬆️   | upstream         |
+| 🏃   | in progress      |
+| 💪   | stretch goal     |
+
+
+## Performance
+- ✅ Identify the parts of conda-store responsible for slowing down new environment builds.
+- 🏃 ~40% of environment build time comes from a slow call to conda-lock, which solves each environment twice (once for conda packages, and again to support pip packages with a vendored version of poetry). The dependency on conda-lock is to be eliminated by implementing official conda support for lockfile generation.
+- ⬆️🏃 Teach conda create  how to handle environment.yml files (#14113)
+- ⬆️🏃 Add support for conda [env] hooks (#13833).
```

Comment:
The links to the issues/PRs need fixing - these are no conda-store issues or PRs